django-web-app/cogniguard/settings.py

```diff
@@ -151,33 +111,17 @@
     messages.ERROR: 'danger',
 }
 
-# email backend configuration
```

Comment:
why have you removed email configuration, 
See the latest file of main branch
fix it